frontend/TestRun/EventsTab.svelte

```diff
@@ -398,7 +539,10 @@
                     {event}
                     similars={event.similars}
                     duplicates={getNumberOfDuplicates(event)}
+                    duplicatesVisible={areDuplicatesVisible(event)}
+                    duplicateParent={event.duplicateParent}
                     {toggleDuplicates}
```

Comment:
Especially considering the function is passed there, so the key could be passed in the arguments when calling this function